Feature/user identifier extension (#143)
vtex-apps/login: CHANGELOG.md

## [Unreleased]

### Added

> - Add `providerPasswordButtonLabel`, `hasIdentifierExtension`, `identifierPlaceholder`, `invalidIdentifierError` configurations to Store Front, which can be edited by the site-editor.

Reviewer: ```suggestion
- Add `providerPasswordButtonLabel`, `hasIdentifierExtension`, `identifierPlaceholder`, `invalidIdentifierError` configurations to Store Front that can be edited in site-editor.
```
